pwiz_tools/Skyline/Model/Results/DocNodeChromInfo.cs

```diff
@@ -525,9 +518,11 @@ public bool IsGoodPeak(bool integrateAll)
 
         public float? GetRatio(int index)
         {
-            return index != RATIO_INDEX_GLOBAL_STANDARDS
-                ? _ratios[index]
-                : _ratios[_ratios.Count - 1];
+            // TODO: RatioIndex

```

Comment:
Not clear what the TODO is about. Clarify and make TODO(nicksh). Also, why keep the ternary expression commented out?